jwst_gtvt/scripts/jwst_gtvt.py

```diff
@@ -27,40 +28,50 @@
 from jwst_gtvt.display_results import display_results
 from docopt import docopt
 from jwst_gtvt.jwst_tvt import Ephemeris
-from jwst_gtvt.plotting import plot_visibility
+from jwst_gtvt.plotting import plot_visibility, plot_interactive_visibility
 
 
 def main(args):
-    if args['--start_date'] and args['--end_date']:
-        start = Time(args['--start_date'])
-        end = Time(args['--end_date'])
+    if args["--start_date"] and args["--end_date"]:
+        start = Time(args["--start_date"])
+        end = Time(args["--end_date"])
         if start > end:
             raise ValueError("Date mismatch, end date is before start date.")
         else:
             eph = Ephemeris(start_date=start, end_date=end)
     else:
         eph = Ephemeris()
 
-    eph.get_fixed_target_positions(args['--ra'], args['--dec'])
+    eph.get_fixed_target_positions(args["--ra"], args["--dec"])
 
-    if not eph.dataframe['in_FOR'].any():
-        in_FOR_msg = ("No position angles in field of regard! "
-                      "Check constraints for your target and if it is observable with JWST. \n"
-                      "Vist: https://jwst-docs.stsci.edu/jwst-observatory-characteristics/jwst-observatory-coordinate-system-and-field-of-regard for more information")
+    if not eph.dataframe["in_FOR"].any():
+        in_FOR_msg = (
+            "No position angles in field of regard! "
+            "Check constraints for your target and if it is observable with JWST. \n"
+            "Vist: https://jwst-docs.stsci.edu/jwst-observatory-characteristics-and-performance/jwst-target-viewing-constraints/jwst-field-of-regard-for#gsc.tab=0 for more information"
+        )
         raise IndexError(in_FOR_msg)
 
-    if args['--write_ephemeris']:
-        args['--silent'] = True
-        eph.write_ephemeris(eph.dataframe, args['--write_ephemeris'])
-
-    if not args['--silent']:
-            display_results(eph)
-
-    plot_visibility(eph, args['--instrument'], name=args['--target_name'], write_plot=args['--write_plot'])
+    if args["--write_ephemeris"]:
+        args["--silent"] = True
+        eph.write_ephemeris(eph.dataframe, args["--write_ephemeris"])
 
+    if not args["--silent"]:
+        display_results(eph)
 
+    if args["--interactive"]:
+        plot_interactive_visibility(
+            eph, args["--instrument"], name=args["--target_name"]
+        )
+    else:
+        plot_visibility(
+            eph,
+            args["--instrument"],
+            name=args["--target_name"],
+            write_plot=args["--write_plot"],
```

Comment:
We should update the keyword argument name here to stay consistent with the recent change in `plotting.py`. Probably worth renaming the CLI flag as well (e.g., --write_name instead of --write_plot).

```suggestion
            write_name=args["--write_plot"],
```

Reply:
@ye-spencer I went back to using `write_plot` instead. Intuitively `write_plot` reads as a `bool` type rather than a `str` type. But I think for the scope of this PR I will reset it since we are already moving some of the plotting code around.